frontend/views.py

```diff
@@ -65,3 +70,40 @@ def get_page(request, file_name):
 
 def redirect_images_to_media(request, image):
     return redirect(settings.MEDIA_URL + "images/" + image)
+
+def submit_page_change(request):
+
+    if request.method == 'POST':
+
+        patch = get_diff(request.POST.get("page", ""),
+                         request.POST.get("content", ""))
+        form = PageEditForm({"page": request.POST.get("page", ""),
+                             "patch": patch,
+                             "comment": request.POST.get("descr", ""),
+                             "email": request.POST.get("email", ""),
+                             "date": datetime.datetime.now()})
+        if form.is_valid():
+            form.save()
+            return HttpResponse('')
+        else:
```

Comment:
useless else

Reply:
Uh you mean the next one ? This one is to report errors like "This is not a valid email"

Reply:
Ah no, since you have a return, you don't have to do a else since it will stop after the end of the if, you can de-indent the next line.